spec/takeaway_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'takeaway.rb'
+
+describe Takeaway do
+
+  let(:customer){ double:customer }
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.